.github/workflows/test_integration_with_coverage.yaml

```diff
@@ -1,6 +1,14 @@
-name: Integration Tests
+name: Integration Tests & Coverage
 
-on: [pull_request]
+on:
+  pull_request:
+    branches:
+      - 'main'
```

Comment:
optional

```suggestion
      - '*'
```

Reply:
For the moment, I would like to only run this for main